src/pages/evm/staking/WithdrawRaw.vue

```diff
@@ -0,0 +1,329 @@
+
+export default defineComponent({
+    name: 'WalletBalanceRow',
+    components: {
+        InlineSvg,
+        ToolTip,
+    },
+    emits: ['update-rex-data'],
+    data() {
+        return {
+            now: Date.now(),
+            pendingTime: '',
+            timer: setTimeout(() => {}, 0),
+        };
+    },
+    props: {
+        withdrawal: {
+            type: Object as PropType<EvmRexDeposit>,
+            required: true,
+        },
+    },
+    computed: {
+        epoch() {
+            return this.withdrawal.until.toNumber();
+        },
+        longDate(): string {
+            return getFormatedDate(this.epoch, 'MMM d, yyyy hh:mm a', false);
+        },
+        isWithdrawable(): boolean {
+            return this.epoch <= this.now / 1000;
+        },
+        token() {
+            const settings = chainStore.getChain(CURRENT_CONTEXT).settings as EVMChainSettings;
+            const token = settings.getSystemToken();
+            return token;
+        },
+        tokenBalanceFiat(): number | null {
+            return this.token.price.isAvailable ? +this.token.price.getAmountInFiatStr(this.withdrawal.amount) : null;
+        },
+        truncatePrimaryValue(): boolean {
+            const isMobile = this.$q.screen.lt.sm;
+
+            if (!isMobile) {
+                return false;
+            } else {
+                return this.primaryAmount.toString().length > 8;
+            }
+        },
+        truncateSecondaryValue(): boolean {
+            const isMobile = this.$q.screen.lt.sm;
+
+            if (!isMobile) {
+                return false;
+            } else {
+                return this.secondaryAmount.toString().length > 7;
+            }
+        },
+        primaryAmount(): number | string {
+            return this.tokenBalanceFiat as number;
+        },
+        prettyPrimaryAmount(): string {
+            return prettyPrintCurrency(
+                +this.primaryAmount,
+                2,
+                fiatLocale,
+                this.truncatePrimaryValue,
+                fiatCurrency,
+                false,
+            );
+        },
+        secondaryAmount(): number | string {
+            let amount = '0';
+            const amountBn = this.withdrawal.amount;
+            amount = ethers.utils.formatUnits(amountBn, this.token.decimals);
+            return amount;
+        },
+        prettySecondaryAmount(): string {
+            const noSecondaryAmount = typeof this.secondaryAmount === 'string' && !this.secondaryAmount;
+            if (noSecondaryAmount) {
+                return '';
+            }
+
+            if (this.truncateSecondaryValue) {
+                return prettyPrintCurrency(+this.secondaryAmount, 4, fiatLocale, true).concat(` ${this.token.symbol}`);
+            } else {
+                const formatted = prettyPrintCurrency(+this.secondaryAmount, 4, fiatLocale);
+
+                return `${formatted} ${this.token.symbol}`;
```

Comment:
`return prettyPrintCurrency(+this.secondaryAmount, 4, fiatLocale, this.token.symbol);`

Reply:
same